### PR TITLE
[MODORDERS-803] - Add test for flow 3

### PR DESCRIPTION
**Pull request by Dmitriy-Butramyou**
## Purpose
_Describe the purpose of the pull request._

## Approach
_How does this change fulfill the purpose?_

### TODOS and Open Questions
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.

### Learning
_Describe the research stage. Add links to blog posts, patterns, libraries or addons used to solve this problem._

### CODE REVIEW

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/update_fund_in_poline_when_invoice_approved.feature [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    When method GET
+    Then status 200
+    * def poLine = $
+    * set poLine.fundDistribution[0].fundId = fundId2
+    * set poLine.fundDistribution[0].code = fundId2
+
+    Given path 'orders/order-lines', poLineId
+    And request poLine
+    When method PUT
+    Then status 403
```
Comment (SerhiiNosko) [inline acquisitions/src/main/resources/thunderjet/mod-orders/features/update_fund_in_poline_when_invoice_approved.feature:151]: Can we check also error code?

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/update_fund_in_poline_when_invoice_approved.feature [hunk 2]
```diff
@@ -0,0 +1,151 @@
+@parallel=false
+Feature: Should fail updating fund in poLine when related invoice is approved
```
Comment (SerhiiNosko) [inline acquisitions/src/main/resources/thunderjet/mod-orders/features/update_fund_in_poline_when_invoice_approved.feature:2]: Please provide in some place name of jira ticket to connect this test accordingly

Comment (Dmitriy-Butramyou) [reply acquisitions/src/main/resources/thunderjet/mod-orders/features/update_fund_in_poline_when_invoice_approved.feature:3]: Added

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/update_fund_in_poline_when_invoice_approved.feature (hunk 1 above)
Comment (Dmitriy-Butramyou) [reply acquisitions/src/main/resources/thunderjet/mod-orders/features/update_fund_in_poline_when_invoice_approved.feature:152]: Added error code check

**Review by SerhiiNosko — APPROVED** [review]

**Review by Abdulkhakimov — APPROVED** [review]

**Pull request merged**